vizro-core/src/vizro/static/js/models/dashboard.js

```diff
@@ -1,3 +1,35 @@
 export function _update_dashboard_theme(checked) {
   return checked ? "vizro_light" : "vizro_dark";
 }
+
+export function _collapse_nav_panel(n_clicks, is_open) {
+  if (!n_clicks)
```

Comment:
Is this the case that is relevant when you change page? Why does it need a separate `if` from the below one?

Reply:
This was relevant when you change page, but I changed the icons and rotation which makes it redundant now. Good catch, I removed it now.